### PR TITLE
refactor: delete all landing reference

### PR DESCRIPTION
**Pull request by leleOFA**

### CODE REVIEW

File: cd-cli/deployShowcaseSite.sh
[the remark this answers is not in the document]
Comment (leleOFA) [reply cd-cli/deployShowcaseSite.sh:1]: done

File: ci/infra/root.yaml [hunk 1]
```diff
@@ -1198,12 +1198,10 @@ Resources:
         CiArtifactBucket: !Ref "CiArtifactBucket"
         NotificationSNSTopic: !Ref "NotificationSNSTopic"
       TimeoutInMinutes: 5
-  PnLandingWebapp:
```
[the remark this answers is not in the document]
Comment (AndreaCimini90) [reply ci/infra/root.yaml:1201]: you have also removed PnLoginWebapp

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: ci/infra/root.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (leleOFA) [reply ci/infra/root.yaml:1201]: revert login

File: cd-cli/deployFrontend.sh [hunk 2]
```diff
@@ -698,8 +649,6 @@ if ( [ ! -z "$HAS_MONITORING" ]) then
         PFTooManyRequestsAlarmArn="${webappPfTooManyRequestsAlarmArn}" \
         PFLoginTooManyErrorsAlarmArn="${webappPflTooManyErrorsAlarmArn}" \
         PFLoginTooManyRequestsAlarmArn="${webappPflTooManyRequestsAlarmArn}" \
-        LandingTooManyErrorsAlarmArn="${landingTooManyErrorsAlarmArn}" \
-        LandingTooManyRequestsAlarmArn="${landingTooManyRequestsAlarmArn}" \
```
Comment (fdammassa) [inline cd-cli/deployFrontend.sh:702]: @AndreaCimini90 vedi commento di sopra

**Review by fdammassa — CHANGES_REQUESTED** [review]

**Review by fdammassa — APPROVED** [review]

**Pull request merged**